pkgs/development/libraries/gstreamer/good/default.nix

```diff
@@ -45,7 +45,7 @@
 , enableDocumentation ? stdenv.hostPlatform == stdenv.buildPlatform, hotdoc
 }:
 
-assert raspiCameraSupport -> (stdenv.isLinux && stdenv.isAarch64);
+assert raspiCameraSupport -> (stdenv.isLinux && stdenv.isAarch32);
```

Comment:
Actually this is a bit of a minor nit, but do you think you add a comment to this linking to the MMAL issue?

That way it's clear why it won't work on aarch64 even when the upstream gstreamer meson file does imply it is supported on aarch32 and aarch64